keepassxc-browser/common/sites.js

```diff
@@ -101,6 +101,16 @@ kpxcSites.exceptionFound = function(identifier, field) {
     return false;
 };
 
+// Forbids using Shadow DOM query with some sites unless a login dialog has been identified
+kpxcSites.isShadowDomQueryAllowed = function(nodeName) {
+    if (document.location.href?.startsWith('https://www.reddit.com')
```

Comment:
Does reddit have relevant sub domains? I know this doesn't break the old reddit.

Reply:
AFAIK the login page always redirects to the new side, even if old.reddit.com is used.